carbon-design-system/carbon
# feat(multiselect): allow disabled listbox items

Closes #9610 
Refs #6477 

Modifies MultiSelect to pass through the `disabled` option on items to downshift. Adds new styles to properly display disabled items.

#### Changelog

**New**

- multiselect: added listbox disabled item styles

**Changed**

- multiselect: enable `disabled` option for listbox items

#### Testing / Reviewing

- View the `carbon-components-react` storybook default MultiSelect story. The third option in the listbox should have the `disabled` attribute and appear disabled.
- View the `carbon-react-next` storybook default MultiSelect story. The third option in the listbox should have the `disabled` attribute and appear disabled.
- Would be awesome to get a visual review to make sure the disabled styles I added are good
